RetailCoder.VBE/UI/CodeExplorer/Commands/AddUserFormCommand.cs

```diff
@@ -32,20 +32,20 @@ protected override bool CanExecuteImpl(object parameter)
 
         protected override void ExecuteImpl(object parameter)
         {
+            var folderAnnotation = $"'@Folder(\"{GetFolder(parameter)}\")";
```

Comment:
We need to figure out a way to have this work when the module is added via standard VBE commands too... somehow.